src/pages/partners/affiliate-ib/_style.js

```diff
@@ -0,0 +1,207 @@
+    position: relative;
+    font-size: 14px;
+    font-weight: normal;
+    color: var(--color-black-3);
+
+    :first-child {
+        margin: 0 2.4rem 0 0;
+    }
+    :last-child {
+        margin: 0 0 0 2.4rem;
+    }
+
+    @media (max-width: 340px) {
+        margin: 0 1.4rem;
+
+        :first-child {
+            margin: 0 1.6rem 0 0;
+        }
+        :last-child {
+            margin: 0 0 0 1.6rem;
+        }
+    }
+`
+export const FormulaValueMobileOneLine = styled.div`
+    display: inline-block;
+    text-align: center;
+    margin: 0 0.8rem;
+    padding: 0.8rem 0;
+    position: relative;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 21px;
+    color: var(--color-black-3);
+
+    :first-child {
+        margin: 0 0.8rem 0 0;
+    }
+    :last-child {
+        margin: 0 0 0 0.8rem;
+    }
+`
+export const FormulaValueResultSwapSynthetic = styled(Flex)`
+    flex-direction: row;
+    text-align: center;
+    font-size: 14px;
+    line-height: 21px;
+    font-weight: normal;
+`
+export const FormulaValueSwapSynthetic = styled.div`
+    display: inline-block;
+    text-align: center;
+    font-size: 14px;
+    line-height: 21px;
+    color: var(--color-black-3);
+    font-weight: normal;
+    margin-top: 0.8rem;
+    margin-left: 1.8rem;
+    margin-right: 1.8rem;
+`
+export const PointerContainer = styled.div`
+    display: flex;
+    margin-top: 8px;
+    flex-direction: column;
+    align-items: center;
+    margin-left: ${(props) => (props.ml ? props.ml : '0')};
+
+    @media ${device.tabletL} {
+        position: absolute;
+        margin-top: ${(props) => (props.top ? '0' : '8px')};
+        flex-direction: ${(props) => (props.top ? 'column-reverse' : 'column')};
+        top: ${(props) => (props.top ? '-16px' : 'unset')};
+        margin-left: ${(props) => (props.top ? '0' : '20px')};
+        left: ${(props) => (props.top ? '50%' : 'unset')};
+        transform: ${(props) => (props.top ? 'translate(-50%, -50%)' : 'unset')};
+    }
+`
+export const PointerContainerMobile = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-left: ${(props) => (props.ml ? props.ml : '0')};
+`
+export const PointerDot = styled.div`
+    height: 5px;
+    width: 5px;
+    border: 1px solid black;
+    border-radius: 50%;
+    opacity: 0.1;
+`
+export const PointerStick = styled.div`
+    position: relative;
+    height: 16px;
+    width: 1px;
+    background-color: black;
+    opacity: 0.1;
+`
+export const PointerText = styled.div`
+    font-size: 14px;
+    font-weight: normal;
+    color: var(--color-grey-5);
+    position: absolute;
+    margin-top: 20px;
+
+    @media ${device.tabletL} {
+        width: 110px;
+        margin: ${(props) => (props.top ? '0 0 26px' : '26px 0 0')};
+        right: ${(props) => (props.top ? '-8px' : 'unset')};
+        left: ${(props) => (props.top ? 'unset' : '-20px')};
+        text-align: ${(props) => (props.top ? 'right' : 'left')};
```

Comment:
Suggestion to name this from `top` to `is_top` since it is used as a boolean toggle 🙏🏼